---
layout: single
permalink: /group/journal_club
title: "Random Number Generator Bioinformatics Journal Club"
toc: true
toc_label: "Table of Contents"
toc_icon: "book-reader"
toc_sticky: true
---

## Random Number Generator Bioinformatics Journal Club

We host the only bioinformatics journal club in the West of Scotland where the topics are chosen by a Random Number Generator!

The Journal Club meets on the second Tuesday of each month (with the occasional move to accommodate other events) and all are welcome! The club usually lasts for 60-90 minutes.

While there are COVID restrictions in place (and it remains convenient) the journal club will be held over Zoom, but the baseline expectation when this ends is that SIPBS-CompBiol group members should be present in person for these meetings.
{: .notice--primary}

The Zoom link and password can be obtained from the SIPBS newsletter, or the calendar invitation. If you can't find it, please contact Leighton.
{: .notice--danger}

### How does it work?

It mostly works like a normal journal club - we all (at least try to) read the paper before the journal club, the speaker introduces a paper, and we discuss it. **But…**

1. At the end of each session, a new speaker volunteers.
2. The speaker rolls a die _once_ to choose a topic from the _current topic pool_. This will be the topic of the paper they present in the next session.
3. The speaker for next time does not need to announce their paper immediately.
4. Once a topic is rolled, it is removed from the list and placed in the _holding pool_.
5. The speaker for next time suggests a new topic for the _current topic pool_ to replace the one they picked.

Suggested topics can be drawn from the holding pool, or newly provided by the speaker, but cannot feature two clubs in succession.
{: .notice--primary}

## Next Journal Club

| Date                 | Location                 | Topic   | Speaker            | Paper  |
|:--------------------:|:------------------------:|:-----:|:------------------:|:-------|
| Tue 13th September 2022   | Zoom (see calendar link or Slack) | Comparative Genomics | Geraldine Buitrago | TBA |

### Current Topic Pool

1. Database
2. Systems Biology
3. Phylogenetic Trees
4. Forensic Science
5. Gene Prediction
6. Protein Structure

## Previous Papers

| Date                 | Speaker | Topic       | Paper  |
|:--------------------:|:------------------------:|:------------------:|:-------|
| 2020-10-11 | Leighton Pritchard | Phylogenomics | Wittouck _et al._, ‘A Genome-Based Species Taxonomy of the Lactobacillus Genus Complex’ [doi:10.1128/mSystems.00264-19](https://doi.org/) |
| 2020-11-10 | Leighton Pritchard | Software Tool | Ondov, B.D., Treangen, T.J., Melsted, P. _et al._ "Mash: fast genome and metagenome distance estimation using MinHash." [doi:10.1186/s13059-016-0997-x](https://doi.org/10.1186/s13059-016-0997-x) |
| 2020-12-08 | Emma Hobbs | Classic Paper (pre-2010) | Thompson _et al._ "`CLUSTAL W`: improving the sensitivity of progressive multiple sequence alignment…" [doi:10.1093/nar/22.22.4673](https://doi.org/10.1093/nar/22.22.4673) |
| 2020-12-08 | Morgan Feeney | Function Prediction | Nie _et al._ "Genomic reconstruction of σ54 regulons in Clostridiales" [doi:10.1186/s12864-019-5918-4](https://doi.org/10.1186/s12864-019-5918-4) |
| 2021-01-12 | Angelika Kiepas | Database | Griffen _et al._ "`CORE`: A Phylogenetically-Curated 16S rDNA Database of the Core Oral Microbiome" [doi10.1371/journal.pone.0019051:](https://doi.org/10.1371/journal.pone.0019051) |
| 2021-01-12 | Nick Tucker | Protein Structure | Stiffler _et al._ "Protein Structure from Experimental Evolution" [doi:10.1016/j.cels.2019.11.008](https://doi.org/10.1016/j.cels.2019.11.008) |
| 2021-02-09 | Ally Hughes | Conserved Domains | Seibt _et al._ "The conserved 3′ Angio‐domain defines a superfamily of short interspersed nuclear elements (SINEs) in higher plants" [doi:10.1111/tpj.14567](https://doi.org/10.1111/tpj.14567) |
| 2021-02-09 | Leighton Pritchard | Protein Evolution | Yu _et al._ "Grammar of protein domain architectures" [doi:10.1073/pnas.1814684116](https://doi.org/10.1073/pnas.1814684116) |
| 2021-03-09 | Emma Hobbs | Algorithm | Xu _et al._ `"eCAMI`: simultaneous classification and motif identification for enzyme annotation" [doi:10.1093/bioinformatics/btz908](https://doi.org/10.1093/bioinformatics/btz908) |
| 2021-03-09 | Morgan Feeney | Antimicrobal Resistance | Laskaris _et al._ "Coevolution of antibiotic production and counter-resistance in soil bacteria" [doi:10.1111/j.1462-2920.2009.02125.x](https://doi.org/10.1111/j.1462-2920.2009.02125.x) |
| 2021-04-13 | Angelika Kiepas | Metabolism | Medema _et al._ "`antiSMASH`: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences" [doi:10.1093/nar/gkr466](https://doi.org/10.1093/nar/gkr466) |
| 2021-05-10 | Leighton Pritchard | Preprint | Hall _et al._ "Gene-gene relationships in an _Escherichia coli_ accessory genome are linked to function and mobility" [doi:10.1093/nar/gkr466](https://doi.org/10.1093/nar/gkr466) |
| 2021-06-08 | Emma Hobbs | Gene Prediction | Zhang _et al._ "Gene Prediction in Metagenomic Fragments with Deep Learning" [doi:10.1155/2017/4740354](https://doi.org/10.1155/2017/4740354) |
2021-07-13 | Morgan Feeney | Application Note | Ducret _et al._ "`MicrobeJ`, a tool for high throughput bacterial cell detection and quantitative analysis" [doi:10.1038/nmicrobiol.2016.77](https://doi.org/10.1038/nmicrobiol.2016.77) |
| 2021-08-10 | Bailey Harrington | Metagenomics | Pechal _et al._ "The potential use of bacterial community succession in forensics as described by high throughput metagenomic sequencing" [doi:10.1007/s00414-013-0872-1](https://doi.org/10.1007/s00414-013-0872-1) |
| 2021-11-09 | Angelika Kiepas | Protein Structure | Jumper _et al._ "Highly accurate protein structure prediction with `AlphaFold`" [doi:10.1038/s41586-021-03819-2](https://doi.org/10.1038/s41586-021-03819-2) |
| 2021-12-14 | Emma Hobbs | Forensic Science | Garafutdinov _et al._ "A new digital approach to SNP encoding for DNA identification" [doi:10.1016/j.forsciint.2020.110520](https://doi.org/10.1016/j.forsciint.2020.110520) |
| 2022-02-08 | Morgan Feeney | Environmental Microbiology | Lalla _et al._ "Winogradsky columns as a strategy to study typically rare microbial eukaryotes" [doi:10.1016/j.ejop.2021.125807](https://doi.org/10.1016/j.ejop.2021.125807) |
| 2022-03-08 | Angelika Kiepas | Classic paper (pre-2010) | Altschul _et al._ "Gapped `BLAST` and `PSI-BLAST`: a new generation of protein database search programs" [doi:10.1093/nar/25.17.3389](https://doi.org/10.1093/nar/25.17.3389) |
| 2022-05-10 | Geraldine Buitrago | Artificial Intelligence | Gardiner _et al._ "Combining explainable machine learning, demographic and multi-omic data to inform precision medicine strategies for inflammatory bowel disease." [doi:10.1371/journal.pone.0263248](https://doi.org/10.1371/journal.pone.0263248) |
| 2022-06-14 | Steve Ford | Metabolism | Sankarasubramanian _et al._ "Gut Microbiota and Metabolic Specificity in Ulcerative Colitis and Crohn's Disease" [doi:10.3389/fmed.2020.606298](https://doi.org/10.3389/fmed.2020.606298) |
| 2022-07-12 | Leighton Pritchard | Software Tool | Kosakovsky Pond _et al._ "HyPhy: hypothesis testing using phylogenies" [doi:10.1093/bioinformatics/bti079](https://doi.org/10.1093/bioinformatics/bti079) |
|2022-08-09| Morgan Feeney|Parasitology| Will et al. "Genetic Underpinnings of Host Manipulation by Ophiocordyceps as Revealed by Comparative Transcriptomics" [10.1534/g3.120.401290](https://doi.org/10.1534/g3.120.401290)




## Holding Pool

- Phylogenomics
- Software Tool
- Classic Paper (pre-2010)
- Function Prediction
- Database
- Protein Structure
- Conserved Domains
- Protein Evolution
- Antimicrobial Resistance
- Algorithm
- Metabolism
- Preprint
- Gene Prediction
- Application Note
- Metagenomics
- Forensic Science
- Artificial Intelligence
- Software Tool
- Parasitology
- Comparative Genomics