---
title: "Development of Tn-Seq Methods for the Functional Characterization of Bacteria"
header:
  overlay_color: "#000"
  overlay_filter: "0.3"
  overlay_image: /assets/images/Transposon_insertion_in_Tn-seq.png
  teaser: /assets/images/Transposon_insertion_in_Tn-seq.png
  og_image: /assets/images/Transposon_insertion_in_Tn-seq.png
  caption: "[Photo Credit: *Wiki User Hillaryandnacef*](https://commons.wikimedia.org/wiki/File:Transposon_insertion_in_Tn-seq.png)"
excerpt: "Enabling the rapid identification of all lethal gene pairs in an organism"
categories:
  - projects
  - in progress
tags:
  - Transposon
  - compositional data Analysis
  - genotyping
---


## Collaborators

* Chris Reisch, University of Florida, Department of Microbiology and Cell Science, Gainesville, FL

## Project

Our understanding of the physiology and metabolism of bacteria is based upon our knowledge of gene function and even well-studied bacteria possess 30-50% of genes with unknown or poorly characterized function. Similarly, the number and identity of essential genes in most model bacteria are unknown. To investigate gene function and essentiality, methods such as Transposon Sequencing (TnSeq) have been developed over the past few years. These methods utilize transposon mutant libraries in which the host organismâ€™s genome has been randomly disrupted by a transposon-antibiotic resistance marker. Using next-generation high-throughput sequencing, the locations of these transposons can be precisely mapped. Genes that are underrepresented or absent from the library can be considered essential. Using different growth conditions and medium formulations yields a subset of genes that are essential only under given conditions, and others that are universally essential. Thus, by comparing the essentiality of genes under different conditions we can begin to elucidate the function of poorly understood genes. Applying this methodology to bacteria that are potential plant pathogens could enable high-throughput identification of genes required virulence, by comparing strains to their non-pathogenic relatives. In addition, by applying this methodology to enteric bacteria we may be able to develop better containment strategies. This project will develop methods for construction of TnSeq libraries and build a bioinformatics pipeline to map, and determine the fitness of mutants using sequencing data.
